testsuite/features/support/env.rb

```diff
@@ -100,6 +100,14 @@ def enable_assertions
   skip_this_scenario unless $proxy
 end
 
+Before('@sle_client') do
```

Comment:
:open_mouth:  That was missing?
Thanks to fix it.

Reply:
Yeah, can happen... No worries.